torch/csrc/CudaIPCTypes.cpp

```diff
@@ -0,0 +1,51 @@
+#include <torch/csrc/CudaIPCTypes.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+CudaIPCReceivedData::CudaIPCReceivedData(std::shared_ptr<void> shared_ptr)
+    : shared_ptr_(std::move(shared_ptr)) {}
+
+int64_t CudaIPCSentData::get() {
+  return *(int64_t*)(shared_storage_ptr_.get());
+}
+
+CudaIPCSentDataLimbo::~CudaIPCSentDataLimbo() {
+  collect();
+  if (shared_blocks_.size() > 0) {
+    AT_ERROR(
```

Comment:
I think some of the internal linters will throw a hissy fit at this because you're throwing an exception in the destructor. I don't remember what the best workaround in this situation is; blasting some stuff to `std::cerr` also seems a bit goofy)

Reply:
Agree, already seen linter complaining. Generally not trying to prevent user from shooting his own leg here. More like pointing to closest ambulance. Will replace with stderr.